reports/report_details.py

```diff
@@ -1329,7 +1350,7 @@ def add_transnational_to_regions(regions):
         '2015': {
             'name': '88',
             'title': 'Internet - Selection of News Subjects',
-            'desc': 'Breakdown by sex of reporter',
+            'desc': 'Internet - Selection of News Subjects',
```

Comment:
```suggestion
            'desc': 'Breakdown by sex of reporter',
```

All `80` to `90` descriptions were also fixed @ascii-dev .